### PR TITLE
Add project status section to README.md

### PR DESCRIPTION
**Pull request by lachie83**
As discussion during the weekly project meeting on Friday, June 25 2021. This adds a project status section to make it more clear about the project maturity.

cc @mnm678 @gokarnm

Signed-off-by: Lachlan Evenson <lachlan.evenson@microsoft.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 A current overview of Notary v2, including a collection of requirements & scenarios, framing the scope of the Notary v2 project.
 
+## Project Status
+
+The Notary v2 project is in early development. Please refer to the [milestones](https://github.com/notaryproject/notaryproject/milestones) or [attend](#contributing-&-conversations) the weekly meetings for details on the roadmap.
```
Comment (mnm678) [inline README.md]:
```suggestion
The Notary v2 project is in early development and design documents should not be considered final. Please refer to the [milestones](https://github.com/notaryproject/notaryproject/milestones) or [attend](#contributing-&-conversations) the weekly meetings for details on the roadmap.
```

Comment (mnm678) [reply README.md]: I also left comments on some of the project milestones. The first one was targeted to be done this month, and as we are still working on requirements I think it should be pushed to the end of the year at the earliest.

**Comment by gokarnm** [comment]
LGTM, thanks @lachie83 !

**Review by SteveLasker — APPROVED** [review]
LGTM

**Pull request merged**